MoltenVK/MoltenVK/GPUObjects/MVKBuffer.mm

```diff
@@ -138,9 +137,7 @@
     return false;
 }
 
-#if MVK_MACOS
```

Comment:
Ditto.

Reply:
I chose to leave this one to avoid an unimplemented function declaration in `.h`. Seemed better than also having to add guard code there.